### ISSUE TITLE
Unable to instantiate service com.pushwoosh.PushFcmIntentService

### ISSUE THREAD

**Issue by rafaelfukuda** [Pushwoosh/pushwoosh-android-sdk#79]
Hello!

We just noticed that some of our users are getting this exception:

```
Fatal Exception: java.lang.RuntimeException: Unable to instantiate service com.pushwoosh.PushFcmIntentService: java.lang.ClassNotFoundException: Didn't find class "com.pushwoosh.PushFcmIntentService" on path: DexPathList[[zip file "/data/app/com.lesara.app-2/base.apk"],nativeLibraryDirectories=[/data/app/com.lesara.app-2/lib/arm, /data/app/com.lesara.app-2/base.apk!/lib/armeabi-v7a, /vendor/lib, /system/lib]]
       at android.app.ActivityThread.handleCreateService(ActivityThread.java:3841)
       at android.app.ActivityThread.access$2100(ActivityThread.java:229)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1909)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:148)
       at android.app.ActivityThread.main(ActivityThread.java:7331)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)
Caused by java.lang.ClassNotFoundException: Didn't find class "com.pushwoosh.PushFcmIntentService" on path: DexPathList[[zip file "/data/app/com.lesara.app-2/base.apk"],nativeLibraryDirectories=[/data/app/com.lesara.app-2/lib/arm, /data/app/com.lesara.app-2/base.apk!/lib/armeabi-v7a, /vendor/lib, /system/lib]]
       at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:56)
       at java.lang.ClassLoader.loadClass(ClassLoader.java:511)
       at java.lang.ClassLoader.loadClass(ClassLoader.java:469)
       at android.app.ActivityThread.handleCreateService(ActivityThread.java:3838)
       at android.app.ActivityThread.access$2100(ActivityThread.java:229)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1909)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:148)
       at android.app.ActivityThread.main(ActivityThread.java:7331)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)
```
The SDK version we are using is 5.1.1.

We don't use FCM and I double checked we are following the legacy GCM Integration guide.
Also we have this rule in our proguard file:

`-keep class com.pushwoosh.** { *; }`

Any insights on what might be the issue?

Thank you!

**Comment by wfhm** [comment]
Hi,

Could you please share your app build.gradle?

**Comment by wfhm** [comment]
@rafaelfukuda by any chance did you have time to review my previous comment?

**Comment by chriswiesner** [comment]
Same here.
Using 5.4.2 - but legacy GCM setup

dependencies in build.gradle:

    compile d('pushwoosh') // 5.4.2
	compile d('pushwooshBadge')
	compile d('pushwooshGcm')
	compile d('supportAppCompat') // 26.0.2
	compile d('gmsGCM')` // 11.4.0

**Comment by wfhm** [comment]
@chriswiesner should be fixed with 5.4.4.

Below you can find a list of app build.gradle dependencies used:

```
dependencies {
    implementation 'com.google.firebase:firebase-messaging:11.6.0'

    compile 'com.android.support:appcompat-v7:27.+'
    compile 'com.android.support:support-v4:27.+'
    compile 'com.pushwoosh:pushwoosh:5.4.4'
    compile 'com.pushwoosh:pushwoosh-gcm:5.4.4'
    compile 'com.google.android.gms:play-services-gcm:11.6.0'
}
```

Project build.gradle:

```
buildscript {
    repositories {
        jcenter()
       google()
    }
    dependencies {
        classpath 'com.android.tools.build:gradle:3.0.0'
        classpath 'com.google.gms:google-services:3.1.0'
    }
}

allprojects {
    repositories {
        jcenter()
        google()
    }
}

task clean(type: Delete) {
    delete rootProject.buildDir
}
```
